spec/presenters/document_type_summary_presenter_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe DocumentTypeSummaryPresenter, type: :presenter do
+  context "when api_response has no `anonymous_feedback_counts`" do
+    let(:api_response) {
+      {
+          "document_type" => "smart_answer",
+          "anonymous_feedback_counts" => [],
+      }
+    }
+    subject(:presenter) { DocumentTypeSummaryPresenter.new(api_response) }
+
+    it "should be empty" do
+      expect(presenter).to be_empty
+    end
+
+    it "should have zero size" do
+      expect(presenter.size).to be_zero
+    end
+
+    it "should present api_response's `anonymous_feedback_counts` as an empty array" do
+      expect(presenter).to eq([])
+    end
+  end
+
+  context "when api_response has `anonymous_feedback_counts`" do
+    let(:path_a) { "/path-a" }
+    let(:path_b) { "/path-b" }
+    let(:path_c) { "/path-c" }
+    let(:api_response) {
+      {
+          "document_type" => "smart_answer",
+          "anonymous_feedback_counts" => [
+              { "path" => path_c },
+              { "path" => path_a },
+              { "path" => path_b },
```

Comment:
Minor: This is probably rubocop being weird, but the indentation here is 4 instead of 2 spaces.

Reply:
Hopefully corrected now. It might have been my RubyMine in the end, which was set to 4 space indentation when I started writing this spec. It is now set to 2 space.